onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -7642,5 +7643,87 @@ TEST_F(GraphTransformationTests, GatherToSliceFusion) {
   }
 }
 
+TEST_F(GraphTransformationTests, GatherSliceToSplitFusion) {
+  {
+    auto build_test_case = [&](ModelTestBuilder& builder) {
+      auto* data_arg = builder.MakeInput<float>({{54}});
+      auto* reshape_arg = builder.MakeInput<int64_t>({{4}});
+      auto* reshape_out = builder.MakeIntermediate<float>({{2, 512, 73, 64}});
+      builder.AddNode("Reshape", {data_arg, reshape_arg}, {reshape_out});
+
+      // Create Shape-0 Ops
+      auto* shape_output_0 = builder.MakeOutput();
```

Comment:
why we need shape here? just 1reshape+2gather+1slice is enough to test your optimizer

Reply:
Yes, deleted the shape ops, thank you!